homotopy-web/src/components/rawhtml.rs

```diff
@@ -4,33 +4,36 @@ use web_sys::Element;
 use yew::prelude::*;
 
 #[derive(Debug, Clone, Eq, PartialEq, Properties)]
-pub struct RawHTMLProps {
-  pub inner_html: String,
+pub struct RawHtmlProps {
+    pub inner_html: String,
 }
 
-pub struct RawHTML {
-    props: RawHTMLProps,
+pub struct RawHtml {
+    props: RawHtmlProps,
     node_ref: NodeRef,
 }
 
-impl Component for RawHTML {
+impl Component for RawHtml {
     type Message = ();
-    type Properties = RawHTMLProps;
+    type Properties = RawHtmlProps;
 
     fn create(props: Self::Properties, _: ComponentLink<Self>) -> Self {
-        Self { props, node_ref: NodeRef::default() }
+        Self {
+            props,
+            node_ref: NodeRef::default(),
+        }
     }
 
     fn update(&mut self, _: Self::Message) -> ShouldRender {
         true
     }
 
     fn change(&mut self, props: Self::Properties) -> ShouldRender {
```

Comment:
There's something in `yewtil` which is essentially this function. I don't remember if we're using it in the codebase, but if we are then please use it here too (otherwise it's probably not worthwhile).

Reply:
I don't think we're using it at the moment but it may become necessary when I start working on the WebGL stuff

Reply:
I'll have a look when I start authoring the render pipeline and switch over then if necessary